### PR TITLE
Create class name: Lobar

### LINKED ISSUE #2: Lobar ismizga clas oching

### PR DESCRIPTION
**Pull request by Yodgorovna**
closes #2

### CODE REVIEW

File: pullrecuest/Lobar.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿namespace pullrecuest;
+
+public class Lobar
+{
+    public static void print()
+    {
+        Console.WriteLine(  "tcvgdgsh");
```
Comment (rustambekovich) [inline pullrecuest/Lobar.cs:7]: koviska ichida kop bosh joy qolgan Lobar

File: pullrecuest/Lobar.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    }
+
+    public static void delete()
+    {
+        Console.WriteLine(  "fcgvg");
```
Comment (jumanboyev) [inline pullrecuest/Lobar.cs:12]: Lobar Console.WriteLine da tushinarli text yozing

**Review by rustambekovich — CHANGES_REQUESTED** [review]
yaxshi

**Review by jumanboyev — CHANGES_REQUESTED** [review]

**Review by Bakhtiyorovna — APPROVED** [review]
App

File: pullrecuest/Lobar.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿namespace pullrecuest;
+
+public class Lobar
+{
+    public static void print()
+    {
+        Console.WriteLine("bo'sh joyniyam o'chirdim");
+    }
+
+    public static void delete()
```
Comment (rustambekovich) [inline pullrecuest/Lobar.cs:10]: jud ayaxshi

**Review by rustambekovich — APPROVED** [review]

**Pull request merged**